src/discrete_info_estimators/jackknife_estimator.jl

```diff
@@ -0,0 +1,18 @@
+export JackknifeEstimator
+
+struct JackknifeEstimator{I <: InformationMeasure} <: DiscreteInfoEstimator{I}
+    measure::I
```

Comment:
this doesn't follow the api we have declared, see the docstring of `InformationMeasureEstimator`

(p.s. why is this the only estimator that ends in `Estimator`? Either all or none should end in that, I vote none)